test/com/facebook/buck/android/AndroidPrebuiltAarIntegrationTest.java

```diff
@@ -16,6 +16,10 @@
 
 package com.facebook.buck.android;
 
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.not;
```

Comment:
That would be neat! In the meantime I set up a pre-push hook for myself, so this shouldn't happen anymore in the future.